src/test/java/model/OrderTest.java

```diff
@@ -0,0 +1,69 @@
+package model;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class OrderTest {
+
+    @Test
+    void getId_ShouldReturn_IdOfOrderedProduct() {
```

Comment:
Sehr schöne Schreibweise! ;)
Lesbar, klar und in allen Tests durchgezogen :+1: 

Die _soll_ Konvention finde ich persönlich richtig gut